lib/oauth/error-responder.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+var _ = require('lodash');
+var url = require('url');
+
+var helpers = require('../helpers');
+var forms = require('../forms');
+var resolveStateToken = require('./common').resolveStateToken;
+
+/**
+ * oauthErrorResponder takes an error object and respondes
```

Comment:
Remove `oauthErrorResponder` and capitalize `takes`. Also `respondes` should be `responds`.